cafebabel/tests/test_articles.py

```diff
@@ -13,8 +13,24 @@
 def test_proposal_displays_form(app, client):
     response = client.get('/article/proposal/new/')
     assert response.status_code == 200
-    assert ('<input name=topic'
-            in response.get_data(as_text=True))
+    assert ('<input name=topic' in response.get_data(as_text=True))
+
+
+def test_create_draft_should_display_form(client, editor):
+    login(client, editor.email, 'secret')
+    response = client.get('/article/draft/new/')
+    assert response.status_code == 200
+    assert '<input id=title' in response.get_data(as_text=True)
+
+
+def test_create_draft_should_generate_article(client, editor):
+    login(client, editor.email, 'secret')
+    response = client.post('/article/draft/new/', data={
+        'title': 'Test article',
+        'language': 'en',
+        'summary': 'In short',
+        'body': 'Article body',
+    }, follow_redirects=True)
```

Comment:
Not actually testing the response?

Reply:
This test had been moved to test_drafts. Removed it from test_articles